Sukanta_Roy/progress.md

```diff
@@ -3,78 +3,66 @@
 **Project: Questions+**
 
 Overall Direction:
-- 
-- 
-- 
-
+- Rubric questions: allow weightages for sub questions
+- MCQ questions: support attaching weights to options
+- Calculate more statistics for MCQ questions (Total, average for each student)
+- MSQ questions: support attaching weights to options
+- Calculate more statistics for MSQ questions
+- Ensure the downloaded data contains all data shown in the page.
+- Add a new question template
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Make UI changes for [Rubric questions: allow weightages for sub questions](https://github.com/TEAMMATES/teammates/issues/7224)
+- [ ] Implement the logic for weights feature in rubric questions sub questions
 
 Week 2 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+**Final Semester Exams**
```

Comment:
Since u are going to do final exams this week, I suggest u to come up with a list of questions for the new question template and we can discuss that template in this week so that the week is not wasted.

Reply:
@xpdavid Yes. That would be a good idea. Adding that to the week 2 plan. 

P.S.: 
>I suggest u to come up with a list of questions for the new question template

Actually, I compiled a list of questions and added that to my proposal, can't we use that for the discussion? I have added ~10 questions in my proposal.

Reply:
Are u going to include all of them in a template?

Reply:
Exactly that what I mean. You can come up with a template first (or even several possible templates) so that we can choose and discuss.

Reply:
@xpdavid Okay, sounds good to me.